.github/workflows/ci.yml

```diff
@@ -16,14 +16,6 @@ jobs:
     steps:
       - uses: actions/checkout@v3
 
-      - name: Install Rust toolchain
-        uses: actions-rs/toolchain@v1
-        with:
-          profile: minimal
-          toolchain: stable
-          components: rustfmt, clippy
-          override: true
-
```

Comment:
We actually already check both of these things just a few lines down: this whole block is unnecessary.